scripts/install_environment.sh

```diff
@@ -256,6 +256,11 @@ function additional_configs() {
     sudo sed -ir 's/net.ipv6.conf.all.disable_ipv6[[:blank:]]*=[[:blank:]]*1/net.ipv6.conf.all.disable_ipv6 = 0/g' /etc/sysctl.conf
     sudo sed -i -e '/net.core.somaxconn/d' -e '$a net.core.somaxconn = 2000' /etc/sysctl.conf
     sudo sysctl --load
+
+    if [ $IPXE_BOOT = "true" ]; then
```

Comment:
```IPXE_BOOT: unbound variable```

Reply:
Fixed, thank you!

Reply:
I think it needs to be ``${IPXE_BOOT:-false}`` to handle the default value

Reply:
I'm also okay with opening that port unconditionally, so whichever of those options is totally okay

Reply:
I think the syntax `if [ ${IPXE_BOOT-false} = "true" ]` works too, setting `IPXE_BOOT` to false if it's not set to anything. Is it preferable to use `${IPXE_BOOT:-false}` instead?

Reply:
Not critical
``${VAR:-value}`` is able to cope with empty values as well, but that's not such an important use-case

Reply:
Modified it! Thanks!!